src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -363,7 +363,10 @@ public static function create($uri, $method = 'GET', $parameters = array(), $coo
             case 'POST':
             case 'PUT':
             case 'DELETE':
-                if (!isset($server['CONTENT_TYPE'])) {
+                if (isset($server['content_type']) {
```

Comment:
Agree..  but if it's expects uppercase keys by design it's not really an issue. And passing in a lowercased version somewhere in the core is a bug :)